frontend/src/tests/components/common/__snapshots__/RenderMultiSelect.spec.js.snap

```diff
@@ -68,8 +68,12 @@ ShallowWrapper {
           showSearch={false}
           transitionName="slide-up"
         >
-          <Option />
-          <Option />
+          <Option
+            title={undefined}
```

Comment:
Please either confirm that this is an optional field or update the test data to populate this value

Reply:
This is an optional field